db/migrate/20170818020825_add_tsheets_user_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddTsheetsUserNameToUsers < ActiveRecord::Migration
+  def change
```

Comment:
Thanks, that's a great suggestion! It's too easy for this stuff to get out of control as you say.